backend-tests/tests/test_useradm_enterprise.py

```diff
@@ -122,41 +119,66 @@ def _make_2fa_settings(self, user_statuses):
 
         return s
 
-    def test_enable_disable(self, tenants_users, smtp_server):
+    @pytest.fixture(scope="function")
+    def setup_user_2fa(self, tenants_users) -> User:
         user_2fa = tenants_users[0].users[0]
-        user_no_2fa = tenants_users[0].users[1]
 
         r = self._login(user_2fa)
         assert r.status_code == 200
-        user_2fa_tok = r.text
-
-        # some error scenarios related to invalid 2fa state
-        # before email verification - can't touch settings
-        for on in [True, False]:
-            r = self._toggle_tfa(user_2fa_tok, user_2fa.id, on=True)
-            assert r.status_code == 403
+        user_2fa.token = r.text
+        yield user_2fa
 
-        # /2faqr available only in 'unverified' state
-        r = uadm.with_auth(user_2fa_tok).call("GET", useradm.URL_2FAQR)
+    @pytest.fixture(scope="function")
+    def setup_user_no_2fa(self, tenants_users) -> User:
+        user_no_2fa = tenants_users[0].users[1]
 
+        r = self._login(user_no_2fa)
+        assert r.status_code == 200
+        user_no_2fa.token = r.text
+        yield user_no_2fa
+
+    def test_2fa_cannot_be_enabled_disabled_before_email_verification(
```

Comment:
How is this related to the reduced wait time in the login?
In other words, what are you fixing here?

Reply:
I created separate commit for that.